torch/nn/functional.py

```diff
@@ -33,8 +33,9 @@
     bias: optional bias of shape :math:`(\text{out\_channels})`. Default: ``None``
     stride: the stride of the convolving kernel. Can be a single number or
       a one-element tuple `(sW,)`. Default: 1
-    padding: implicit zero paddings on both sides of the input. Can be a
+    padding: implicit zero or circular paddings on both sides of the input. Can be a
```

Comment:
nit: I'd probably just say "implicit paddings" here and let `padding_mode` explain what the input padding is. Otherwise you have to keep updating this list whenever people add more padding modes.